### PR TITLE
Smaller ui fixes

### PR DESCRIPTION
**Pull request by AtiX**

### CODE REVIEW

File: src/client/landingpage.coffee [hunk 1]
```diff
@@ -88,7 +88,7 @@ b1 = bundle1.init().then ->
 		@value = ''
 
 	$('.dropper').text 'Drop an stl file'
-	$('#preview img').on( 'dragstart'
+	$('#preview img,a').on( 'dragstart'
```
Comment (Steditor) [inline src/client/landingpage.coffee]: Must be `#preview img, #preview a` - we don't want to select all links on the whole page.

File: views/landingpage/teaser.jade [hunk 2]
```diff
@@ -14,31 +14,35 @@ header
...
 				)
 
-			a#educators.btn(href='/educators')
-				#preview
-					img(
-						src='/img/sample_ballista.png',
-						draggable='true',
-						data-hash='708d0b0b44f491523de951afa44b3e06'
-					)
+				#educators
```
Comment (Steditor) [inline views/landingpage/teaser.jade]: Is everything a child of `.drop` on purpose?

**Comment by Steditor** [comment]
:boat:, pls recheck (e.g. @stuikomma ?)

File: routes/landingpage.coffee [hunk 3]
```diff
@@ -4,6 +4,24 @@ module.exports.getLandingpage = (request, response) ->
...
+			},
+			{
+				hash: '1c2395a3145ad77aee7479020b461ddf'
+				img: 'sample_goggles.png'
+			},
+			{
+				hash: '1b2512f0804dfd68b2783108d840ab6f'
+				img: 'sample_printableWrench.png'
+			}
+		]
```
Comment (stuikomma) [inline routes/landingpage.coffee]: We should also use the same method for the educators page.

Comment (Steditor) [inline routes/landingpage.coffee]: :+1:

Comment (Steditor) [inline routes/landingpage.coffee]: Will be done in combination with #508

**Pull request merged**